templates/guides/quickstart-dhcp.md

```diff
@@ -0,0 +1,236 @@
+---
+page_title: "Managing DHCP service with the BloxOne Terraform Provider"
+subcategory: "Guides"
+description: |-
+  This guide provides step-by-step instructions for using the BloxOne Terraform Provider to manage IPAM and DHCP resources.
+---
+
+# Managing DHCP service with the BloxOne Terraform Provider
+
+This guide provides step-by-step instructions for using the BloxOne Terraform Provider to manage IPAM and DHCP resources.
+
+## Configuring the Provider
+
+The provider needs to be configured with an API key and the URL of the Infoblox Cloud Services Portal (CSP). The API Key can be obtained from the Infoblox Cloud Services Portal (CSP) by following the steps in the guide - [Configuring User API Keys](https://docs.infoblox.com/space/BloxOneCloud/35430405/Configuring+User+API+Keys).
+
+Create a directory for the Terraform configuration and create a file named `main.tf` with the following content:
+
+````terraform
+terraform {
+  required_providers {
+    bloxone = {
+      source  = "infobloxopen/bloxone"
+      version = "~> 0.1.0"
+    }
+  }
+}
+
+provider "bloxone" {
+  csp_url = "https://csp.infoblox.com"
+  api_key = "<BloxOne API Key>"
+}
+````
+
+!> Warning: Hard-coded credentials are not recommended in any configuration file. It is recommended to use environment variables.
```

Comment:
is that a valid markdown ?

Reply:
Terraform Registry documentation accepts it as a special case. It shows up as a colored callout box. 

https://developer.hashicorp.com/terraform/registry/providers/docs#callouts